packages/utils/create-react-app/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@parcel/create-react-app",
+  "version": "2.0.0-beta.7",
+  "bin": {
+    "parcel-create-react-app": "lib/bin.js"
```

Comment:
I guess this should be `src` as well, but you'd never run invoke the CLI in to monorepo anyway 🤷